src/patternfly/components/NotificationBadge/notification-badge.scss

```diff
@@ -0,0 +1,52 @@
+.pf-c-notification-badge {
+  --pf-c-notification-badge--after--BorderColor: var(--pf-global--BackgroundColor--100);
+  --pf-c-notification-badge--after--BorderRadius: var(--pf-global--BorderRadius--lg);
+  --pf-c-notification-badge--after--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-notification-badge--after--Top: 0;
+  --pf-c-notification-badge--after--Right: 0;
+  --pf-c-notification-badge--after--Width: calc(var(--pf-global--spacer--sm) + var(--pf-c-notification-badge--after--BorderWidth) + var(--pf-c-notification-badge--after--BorderWidth));
+  --pf-c-notification-badge--after--Height: calc(var(--pf-global--spacer--sm) + var(--pf-c-notification-badge--after--BorderWidth) + var(--pf-c-notification-badge--after--BorderWidth));
+  --pf-c-notification-badge--after--BackgroundColor: transparent;
+
+  // This translation works for the Bell icon. If other icons are used, a modifier may need to be created to allow for proper positioning on that icon.
+  --pf-c-notification-badge--after--TranslateX: calc(var(--pf-global--spacer--xs) + var(--pf-c-notification-badge--after--BorderWidth));
+  --pf-c-notification-badge--after--TranslateY: var(--pf-c-notification-badge--after--BorderWidth);
+
+  // The icon is normalized to be square in order to facilitate consistency in positioning
+  --pf-c-notification-badge--after__i--Width: var(--pf-global--spacer--md);
+  --pf-c-notification-badge--after__i--Height: var(--pf-global--spacer--md);
+
+  // Modifiers
+  --pf-c-notification-badge--m-read--after--BackgroundColor: transparent;
+  --pf-c-notification-badge--m-read--after--BorderColor: transparent;
+  --pf-c-notification-badge--m-unread--after--BackgroundColor: var(--pf-global--active-color--400);
+
+  position: relative;
```

Comment:
looks like we should also add `display: inline-block;` so it wraps the icon properly in FF and Edge.

Reply:
This fixes it for FF but causes Chrome to position the icon slightly differently, so the translation for the badge is off by 2 px. I adjusted it by 1px to create a happy medium between FF and Chrome. If you know some other magic to position it consistently between the browsers, let me know.

Reply:
Does using that in combination with `line-height: 1` improve/normalize the positioning? That should cause the element to wrap the icon better.